packages/crawler/src/authenticator/browser-authenticator.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import Apify from 'apify';
+const { log: apifyLog } = Apify.utils;
+import * as Puppeteer from 'puppeteer';
+
+const attemptAuthentication = async (
+    page: Puppeteer.Page,
+    accountName: string,
+    accountPass: string,
+    logger: typeof apifyLog,
+    attemptNumber: number = 1,
+): Promise<void> => {
+    await page.goto('https://portal.azure.com');
```

Comment:
why this URL is hardcoded here?

Reply:
It's okay to keep it inline if we have concrete implementation for azure provider only under some provider interface generalization.